### ISSUE TITLE
Series.skew() for constant series returns inconsistent values

### ISSUE THREAD

**Issue by yonil7** [pydata/pandas#11974]
I'm not sure if this is a bug, but I'v noticed a strange behavior of skew() and kurt() on constant series:
```python
#skew:
pd.Series(3075.3, np.arange(300)).skew() #returns -105683727.49921818
pd.Series(3075.2, np.arange(300)).skew() #returns nan
pd.Series(3075.5, np.arange(300)).skew() #returns 0

#kurt:
pd.Series(3075.3, np.arange(300)).kurt() #returns 9305221026420888.0
pd.Series(3075.2, np.arange(300)).kurt() #returns -7459707710545691.0
pd.Series(3075.5, np.arange(300)).kurt() #returns 0
```
(using pandas 0.17.1 (from anacondas) on a 64bit intel windows 7 pc)

**Comment by jreback** [comment]
xref for rolling versions of ``kurt/skew``: https://github.com/pydata/pandas/pull/8270

we fixed this for ``var`` here: https://github.com/pydata/pandas/pull/10679

interested in doing a similar fix for ``kurt/skew``?

**Comment by jreback** [comment]
cc @jvkersch

**Comment by jvkersch** [comment]
I can pick this up some time next week, if there are no other takers.

**Comment by jreback** [comment]
@jvkersch that would be great!

**Comment by jreback** [comment]
closed by #12121
